autocannon.js

```diff
@@ -243,6 +243,9 @@ function createChannel (onport) {
 function runTracker (argv, ondone) {
   const tracker = run(argv)
 
+  // While using workers, tracking is handled in manager.js
+  if (argv.useWorkers) return
```

Comment:
I would prefer we move `runTracker` to its own file, and call it or `manager.js` depending if we are using workers or not.

Reply:
Not sure I got it 100%, but updated to move runTracker in a separate file. LMK what you think